blocks/v2-stories-carousel/v2-stories-carousel.css

```diff
@@ -0,0 +1,258 @@
+:root {
+  --stories-carousel-img-height: 342px;
+  --stories-carousel-img-width: 85%;
+  --stories-carousel-gap: 12px;
+}
+
+/* Full width block */
+main .section.v2-stories-container {
+  padding: 0;
+}
+
+main .section.v2-stories-carousel-container .v2-stories-carousel-wrapper {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+/* End Full width block */
+
+.v2-stories-carousel {
+  position: relative;
+}
+
+.v2-stories-carousel ul {
+  list-style: none;
+  padding: 0;
+}
+
+.v2-stories-carousel-items {
+  display: flex;
+  gap: var(--stories-carousel-gap);
+  overflow: scroll hidden;
+  scroll-behavior: smooth;
+  scroll-snap-align: center;
+  scroll-snap-type: x mandatory;
+  scrollbar-width: none;
+}
+
+.v2-stories-carousel-items::-webkit-scrollbar {
+  display: none;
+}
+
+.v2-stories-carousel-item {
+  flex: 0 0 var(--stories-carousel-img-width);
+  scroll-snap-align: center;
+  text-align: center;
+}
+
+.v2-stories-carousel-item:first-child {
+  margin-left: calc(100% - var(--stories-carousel-img-width)/2);
+}
+
+.v2-stories-carousel-item:last-child {
+  margin-right: calc(100% - var(--stories-carousel-img-width)/2);
+}
+
+.v2-stories-carousel-item img {
+  display: block;
+  height: var(--stories-carousel-img-height);
+  object-fit: cover;
+  width: 100%;
+}
+
+.v2-stories-carousel-text {
+  color: var(--c-main-black);
+  font-family: var(--font-family-body);
+  font-size: 1.6rem;
+  letter-spacing: 0.5px;
+  line-height: 130%;
+  opacity: 0;
+  padding: 30px;
+  pointer-events: none;
+  text-align: center;
+  text-wrap: balance;
+  transition: opacity 0.3s;
```

Comment:
Could you add the [duration and easing options mentioned in Volvo Core Design Kit](https://www.volvogroup.com/experiencesystem/en/#/brand/615bfcc428c818001105b4e3/61a0de32a7047974164f01dd/6458984c6c7c54866a028586) to styles.css. Let’s start using them as custom properties for all transitions from now on :)